### PR TITLE
Complete login and sign up

### PR DESCRIPTION
**Pull request by Willylaisa**
basic structure of login page

### CODE REVIEW

File: login.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                            <label for="email">Email: </label>
+                            <input type="email" name="email" id="email" placeholder="E.g john123@gmail.com">
+                        </div>
+                        <div>
+                            <label for="password">Password: </label>
+                            <input type="password" name="password" id="password" placeholder="********">
+                        </div>
+                    </div>
+                    <div class="buttons">
+                        <a href="index.html" class="login-btn">
```
Comment (patrickleon710) [inline login.html]: Is the idea to have a link or a button for the signing/logging in?

File: login.css [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+
+#main-font {
+    font-size: 5.6rem;
+    font-weight: bolder;
+    color: #19A7CE;
+    white-space: nowrap;
+}
+
+@media only screen and (min-width: 768px) {
```
Comment (patrickleon710) [inline login.css]: Shouldn't it be 
     @media screen and (min-width: 768px)   ?

**Review by patrickleon710 — COMMENTED** [review]
* I really like the work. You've used settings I could not remember.
Thumbs up.

File: login.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+</head>
+<body>
+    <section>
+        <div class="welcome">
+            <h2 class="font">Welcome to</h2>
+            <h1 class="font" id="main-font">My Blog</h1>
+        </div> 
+        <div>
+            <form class="login-section">
+                <fieldset>
```
Comment (Brikita) [inline login.html]: What about adding a legend which shows Login?

**Review by Brikita — COMMENTED** [review]
This is great good work.

File: login.html (hunk 1 above)
Comment (Willylaisa) [reply login.html]: I haven't found a way of using JavaScript to make a button to open another page, but please inform me if you haave any idea

File: login.css (hunk 2 above)
Comment (Willylaisa) [reply login.css]: Good point ☝️
Let me check on that

File: login.html (hunk 3 above)
Comment (Willylaisa) [reply login.html]: Great idea .. let me first practise using it😂

**Review by Brikita — APPROVED** [review]
I think it good for now

File: login.html (hunk 1 above)
Comment (patrickleon710) [reply login.html]: You could wrap the anchor tag inside a button tag.

**Review by patrickleon710 — COMMENTED** [review]
I noted there is no username in the form. I thought we had agreed to have both the name and username, please advise.

**Review by itsrandyM — APPROVED** [review]
good work on the login page.

**Pull request merged**